packages/styled-components/src/models/StyledNativeComponent.js

```diff
@@ -39,8 +39,8 @@ function useResolvedAttrs<Config>(theme: any = EMPTY_OBJECT, props: Config, attr
   const context = { ...props, theme };
   const resolvedAttrs = {};
 
-  attrs.forEach(attrDef => {
-    let resolvedAttrDef = attrDef;
+  for (let i = 0; i < attrs.length; i += 1) {
```

Comment:
@anatoliy8493 if you use a for loop you need to use `hasOwnProperty` to check that it belongs to the object and not the prototype. forEach is better here imo.

Reply:
the same as previous